test/services/helpers/services/schools.js

```diff
@@ -28,7 +28,7 @@ const create = (app) => async ({
 		const localSystem = (await app.service('systems').find({ query: { type: 'local' }, paginate: false }))[0];
 		systems.push(localSystem._id);
 	}
-	const school = await School.create({
+	const school = await app.service('schools').create({
```

Comment:
In a other PR, i start mapping at the same time the services to models. To clearify the test cases.

Reply:
The problem is, that there is a lot of setup in the hooks, for example the school years